pyscf/adc/uadc.py

```diff
@@ -3782,16 +3811,16 @@ def ip_compute_trans_moments(adc, orb, spin="alpha"):
             t1_3_a, t1_3_b = adc.t1[1]
 
             if orb < nocc_a:
-                T[s_a:f_a] += 0.25*lib.einsum('kdc,ikdc->i',t2_1_a[:,orb,:,:], t2_2_a, optimize = True)
-                T[s_a:f_a] -= 0.25*lib.einsum('kdc,ikdc->i',t2_1_ab[orb,:,:,:], t2_2_ab, optimize = True)
-                T[s_a:f_a] -= 0.25*lib.einsum('kcd,ikcd->i',t2_1_ab[orb,:,:,:], t2_2_ab, optimize = True)
+                T[s_a:f_a] += 0.25*lib.einsum('kdc,ikdc->i',np.ascontiguousarray(t2_1_a[:,orb,:,:]), t2_2_a, optimize = True)
```

Comment:
We shouldn't apply np.ascontiguousarray in-line. Those should be defined as temporary arrays. Because of dereferencing (specifying orb), these arrays will be small, so this won't be costly in memory. Check if this needs to be done for RADC.

Reply:
Done